website/src/pages/contribute/ContributeCard.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import IconComment from "../../assets/icons/comment.svg";
+
+const Card = styled.div`
+    @media (max-width: 700px) {
+        width: 100%;
+    }
+    width: 80%;
+    border: 1px solid rgb(232, 67, 62);
+    border-radius: 5px;
+    margin-bottom: 1rem;
+    padding: 0.75rem 1.25rem;
+
+    &:hover {
+        background-color: rgb(255,241,240,0.2);
+        cursor: pointer;
+        p{
+            color: rgb(232, 67, 62);
+        }
+    }
+    background-color: ${(props) => (props.isShow ? "rgb(255,241,240,0.2)" : "")}
+`;
+
+const ProjectTitle = styled.div`
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 1.25 rem;
+    margin-right: 1.25 rem;
+`;
+const Title = styled.p`
+  margin: 0;
+  font-size: 1.5rem;
+  color: ${(props) => (props.isShow ? "rgb(232, 67, 62)" : "")}
+`;
+
+const Issue = styled.div`
+    @media (max-width: 700px) {
+        min-width: 5rem;
+    }
+    border: 1px solid rgb(232, 67, 62);
+    border-radius: 0.5rem;
+    padding: 0.25rem 0.5rem;
+    font-size: .875rem;
+`;
+
+const ProjectDesc = styled.div`
+    display: flex;
+    color: ${(props) => (props.isShow ? "rgb(232, 67, 62)" : "")}
+`;
+const List = styled.div`
+    display: ${(props) => (props.isShow ? "block" : "none")};
+`;
+const ListItem = styled.li`
+    list-style: none;
+    display: flex;
+`;
+
+
+const ContributeCard = (props) => {
+    const { repoName } = props;
+    const [isShow, setisShow] = useState(false);
+    const [repoInfo, setRepoInfo] = useState({ description: '', lang: '', stars: '' });
```

Comment:
I feel adding the `lang` attribute here isn't necessary because we can't keep the most used language in a project as a representative of it. What I mean is, one can contribute in the `apisix-dashboard` project even if they are not familiar with Golang, putting the `lang` attribute upfront feels like it is a prerequisite to contribute in that project.

Reply:
good suggestion, I will fix it.